kindle.py

```diff
@@ -280,6 +292,13 @@ def download_books(self, start_index=0, filetype="EBOK"):
         default="EBOK",
         help="to download personal documents or ebook",
     )
+    parser.add_argument(
+            "--max-tries",
+            dest="max_tries",
```

Comment:
I prefer `max_retries` here and in the function arguments

Reply:
Agree

Reply:
In download_one_book function, this is an auto-decrementing variable, so it would be better to use retries,
but max_retries at command line is prefer.